packages/components/src/site-thumbnail/docs/example.jsx

```diff
@@ -1,4 +1,4 @@
-import WordPressLogo from '../../wordpress-logo';
+import { WordPressLogo } from '../../wordpress-logo';
```

Comment:
Those changes are yet another example of something that can be moved to another PR.